core/baseline-core-v1.0-psd01.md

```diff
@@ -3001,60 +3003,227 @@ This section of the document focuses on the requirements and considerations rela
+
+* A test BPI system is available and functional.
+* At least two independent sources of external non-authoritative, non-deterministic are available that produce the same input data.
+* A test workstep has been created that requires external non-authoritative, non-deterministic input data.
+
+Test Steps:
+
+1. Start the test workstep that requires external non-authoritative, non-deterministic input data.
+2. Source the input data from a single external source and record the source of the data.
+3. Repeat step 2, but use a different external source for the same input data.
+4. Repeat step 2, alternating between the external sources used in steps 2 and 3.
+5. Verify that the BPI workstep successfully processed the input data from all sources used in steps 2-4 and produced the same expected result for the input data from different sources.
+
+Test Passing Criteria:
+
+* The test workstep should process input data from all sources used in steps 2-4 without errors or inconsistencies.
+* The expected results from the test workstep should match the actual results produced in step 5.
+
+
+#### **[R315]** 
 External non-authoritative, non-deterministic BPI input data into a BPI workstep MUST be validated by one or more authoritative entities.
 
-[[R316]](#r316) Testability: Agreement on validation of non-authoritative, non-deterministic BPI input data is an extension of non-repudiation of the commercial documents requirement, [[R9]](#r9) which is testable, because non-authoritative, non-deterministic BPI input data sources will be enumerated in a commercial document. Therefore, the requirement is testable. 
+[[R315]](#r315) Testability: 
 
-#### **[R317]** 
-BPI participants MUST agree upon a standard validation method for external non-authoritative,  non-deterministic input data to a BPI workstep.
+Preconditions:
 
-[[R317]](#r317) Testability: Agreement on the validation method of non-authoritative, non-deterministic BPI input data is an extension of non-repudiation of the commercial documents requirement, [[R9]](#r9) which is testable, because non-authoritative, non-deterministic BPI input data sources will be enumerated in a commercial document. Therefore, the requirement is testable.
+* A test BPI system is available and functional.
+* A test workstep that takes external non-authoritative, non-deterministic input data.
+* One or more authoritative entities responsible for validating the input data.
+* The validation criteria defined and agreed upon by all BPI subjects involved in the test workstep.
 
-#### **[R318]** 
+Test Steps:
+
+1. Initialize the test workstep with external non-authoritative, non-deterministic input data from one or more data sources.
+2. Submit the input data to the authoritative entities responsible for validation.
+3. The authoritative entities validate the input data against the predefined validation criteria.
+4. If the input data passes the validation criteria, proceed to the next step. Otherwise, fail the test.
```

Comment:
This should be split up. The first stop is to get to this point, where the input data does not validate according to an authorative entity, and stop.

Then check that is works as expected if everything is right.

Reply:
corrected